app/components/filters/organization-status-toggle.hbs

```diff
@@ -0,0 +1,8 @@
+<div class={{if @error "ember-power-select--error"}}>
```

Comment:
This wrapper isn't needed I think? `@error` isn't passed into the component.

Reply:
Ah yeah good catch